src/DIRAC/Core/Security/m2crypto/X509Chain.py

```diff
@@ -435,7 +437,7 @@ def generateProxyToString(self, lifetime, diracGroup=False, strength=1024, limit
         Args:
             lifetime (int): expected lifetime in seconds of proxy
             diracGroup (str): diracGroup to add to the certificate
-            strength (int): length in bits of the pair if proxyKey not given (default 1024)
+            strength (int): length in bits of the pair if proxyKey not given (default 2048)
```

Comment:
You could use an f-string here:

```suggestion
            strength (int): length in bits of the pair if proxyKey not given (default {DEFAULT_PROXY_STRENGTH})
```

Reply:
f-string cannot be used as docstring as I understand
https://docs.python.org/3/reference/lexical_analysis.html#formatted-string-literals

> Formatted string literals cannot be used as docstrings, even if they do not include expressions.